hapi-fees/src/main/java/com/hedera/services/usage/token/TokenMintUsage.java

```diff
@@ -68,7 +68,7 @@ public FeeData get() {
 		}
 
 		addEntityBpt();
-		addTokenTransfersRecordRb(1, 1);
+		addTokenTransfersRecordRb(1, 1, 1);
```

Comment:
If the token is NON_FUNGIBLE we should do:
```
addTokenTransfersRecordRb(1, 0, op.getMetadatalist());
```

if TOKEN_FUNGIBLE_COMMON we should do:
```
addTokenTransfersRecordRb(1, 1, 0);
```

Reply:
Corrected!